CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 0.1.2
+
+**Update csv generation :**
+
+Much better SQL query and some fixes I have made for making the filters work. We no longer need the frontend fix for the downloads as they are now almost instant.
```

Comment:
No need to be personal in a changelog. We won't know who this `I` was.
Also no need to mention the frontend bit as there's no clue of what that is anywhere else in the changelog or the repo

Reply:
Sorry @Levia you are right I copied and pasted from a PR and edited it, but I forgot to take out the `I`.

Reply:
I have fixed that now @Levia !